frontend/src/modules/venues/components/VenuesTable.tsx

```diff
@@ -186,16 +186,12 @@ export const VenuesTable = ({ refresh }: VenuesTableProps) => {
                   onClick={() => handleView(vn)}
                 >
                   <Td maxW={32}>
-                    <EllipsisText
-                      as='span'
-                      fontWeight='medium'
-                      data-cy='venue-name'
-                    >
+                    <EllipsisText fontWeight='medium' data-cy='venue-name'>
```

Comment:
@noice-noise is it fine that `as='span'` gets removed?

Reply:
Yes, removing the span solves the problem, it's the solution I've provided when jaq raised the issue.